tests/integration_tests/test_game.py

```diff
@@ -532,3 +537,109 @@ async def test_gamestate_ended_clears_references(
 
     assert test.lobby_connection.game_connection is None
     assert rhiza.lobby_connection.game_connection is None
+
+
+@pytest.mark.rabbitmq
+@fast_forward(30)
+async def test_game_ended_broadcasts_army_results(lobby_server, channel):
+    mq_proto_all = await connect_mq_consumer(
+        lobby_server,
+        channel,
+        "success.gameResults.create"
+    )
+
+    host_id, _, host_proto = await connect_and_sign_in(
+        ("test", "test_password"), lobby_server
+    )
+    guest_id, _, guest_proto = await connect_and_sign_in(
+        ("Rhiza", "puff_the_magic_dragon"), lobby_server
+    )
+    await read_until_command(guest_proto, "game_info")
+    ratings = await get_player_ratings(host_proto, "test", "Rhiza")
+
+    # Set up the game
+    game_id = await host_game(host_proto)
```

Comment:
Looks like this helper function already has a parameter for featured mod. You'll just have to add the mod to the `test-data.sql` file.
```suggestion
    game_id = await host_game(host_proto, mod="gw")
```